proxy/envoy/resources.go

```diff
@@ -84,6 +84,13 @@ const (
 	// MONGOProxyFilter is the name of the MONGO Proxy network filter.
 	MONGOProxyFilter = "mongo_proxy"
 
+	// REDISProxyFilter is the name of the REDIS Proxy network filter.
+	REDISProxyFilter = "redis_proxy"
+
+	// REDISDefaultOpTimeout is the op timeout used for REDIS Proxy filter
+	// TODO - Allow this to be configured.
+	REDISDefaultOpTimeout = 30 * time.Second / time.Millisecond
```

Comment:
either use int64 instead or do the conversion when you use it ([line 420](https://github.com/istio/pilot/pull/1396/files#diff-18db4698a9b14dcd564faeefd7fbd74aR420)). the 2nd option is probably better.

Reply:
I am using the second option in line 420. Do you want to change it?

Reply:
`REDISDefaultOpTimeout = 30 * time.Second / time.Millisecond` => 30µs, which is incorrect. I assume you want to set OperationTimeoutMS to 30 seconds, so what I'm recommending is
`REDISDefaultOpTimeout = 30 * time.Second`
and
`OperationTimeoutMS: int64(REDISDefaultOpTimeout / time.Millisecond)` => 30000
Basically, do all the conversion in one place.